canonical/canonical.com
# Create /solutions/telco/5g-edge webpage

## Done

- Created /solutions/telco/5g-edge page
- Similar to the /solutions/telco/5g-core page
  - Minor fixes to this page for the equal height row items

## QA

- Go to https://canonical-com-1383.demos.haus/solutions/telco/5g-edge
- Check that design matches against [Figma](https://www.figma.com/design/brgzRViEncgKKD4KzVnyXy/24.10-Canonical.com%2Fsolutions?node-id=440-6534&node-type=frame&m=dev) and content matches [copy doc](https://docs.google.com/document/d/1K_aHQNapkDgLdCNpLsB1oEeHxo68TMDJoU-iDYlI7lU/edit)

## Issue / Card

Fixes [WD-14131](https://warthogs.atlassian.net/browse/WD-14131)

## Screenshots

[if relevant, include a screenshot]


[WD-14131]: https://warthogs.atlassian.net/browse/WD-14131?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
